bucket/ddc-toolbox.json

```
@@ -0,0 +1,26 @@
+{
+    "version": "1.4",
+    "description": "Create and edit ViPER DDC files, with AutoEQ integration.",
+    "homepage": "https://github.com/ThePBone/DDCToolbox",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ThePBone/DDCToolbox/releases/download/1.4/Portable_DDCToolbox_win64_1.4.zip",
+            "hash": "159c0155f3bcf543c6c554c455c03f7ccbc1d52d29995ff911028f876444eac2"
+        }
+    },
+    "shortcuts": [
```

https://github.com/ThePBone/DDCToolbox/blob/44c8ae5368025dba16e0837476084b72060eaec9/main.cpp#L25-L27

```suggestion
    "bin": "DDCToolbox.exe",
    "shortcuts": [
```